_pages/content-types/forms.md

```diff
@@ -0,0 +1,94 @@
+
+Create labels that are concise and to the point — First name is much easier to comprehend, at a glance, than What moniker do you go by? And, as we noted earlier, short labels are also easier to read on mobile, which improves your form’s accessibility across devices and potentially saves you money on translations.
+
+## Align field length with response length
+
+Field length should correspond to the amount of information you’re requesting. Fields for capturing a first or a last name or a city of residence can be short; fields meant to capture longer responses — a complaint or request, for instance — should necessarily be longer so that people can see what they’re typing before submitting it.
+
+## Don’t use placeholder text as labels
+
+Labels should appear above fields, not as placeholder text inside those fields. Not only can using placeholder text as labels create confusion around which information belongs where (it’s easy to forget what a field is for if it’s not labeled), but it can also make form completion more difficult for people with disabilities. 
+
+## Don’t make optional information required
+
+If a piece of information is optional, label it as such. Requiring only the information that’s truly required will save your users time — something all of us can appreciate. It’s also a good practice to label optional fields instead of labeling required ones; seeing the optional labels, users will infer that the unlabeled fields are for required information. 
+
+## Follow accessibility best practices
+
+If not designed thoughtfully, forms can present major accessibility issues. Inaccessible forms can impact a range of people — screen reader users, keyboard-only users, people with cognitive disabilities, and those who are mobility impaired, to name a few — creating unnecessary barriers. To ensure your forms are accessible, take the following steps.
+
+### Use a proper label element
+
+Associate all data entry fields with labels so that screen reader users can easily tell what type of information they should enter. When data fields are properly labeled, folks using screen readers can complete the form using forms mode. 
+
+The label tag associates a single label tag to a single form element (text field, radio button, checklist item, and so on). Use the for attribute to associate a label with a form element by matching the value of the for attribute to the value of the element's id attribute:
+
+`<form action="/signup" method="post">
+  <label for=”myinput”>Label text</label>
+  <input id=”myinput” name=”textfield” type=”text”>
+</form>`
+
+### Use fieldset and legend for grouping
+
+Use fieldset and legend tags to group related input elements, particularly with sets of radio buttons and checkboxes. Fieldsets display with a border, which is helpful for all users (especially those living with cognitive disabilities). Be sure to accompany all fieldsets with legend tags, which describe the associations:
```

Comment:
> Fieldsets display with a border

You may want to qualify this with "By default, fieldsets display with a border" or something like that.  For example, [USWDS radio buttons in a fieldset](https://standards.usa.gov/components/form-controls/#radio-buttons) don't have a border around them.